contract_manager/scripts/fetch_account_balance.ts

```diff
@@ -0,0 +1,58 @@
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { DefaultStore, PrivateKey, toPrivateKey } from "../src";
+
+const parser = yargs(hideBin(process.argv))
+  .usage("Usage: $0 --private-key <private-key> [--chain <chain>]")
+  .options({
+    "private-key": {
+      type: "string",
+      demandOption: true,
+      desc: "Private key to use to sign transaction",
+    },
+    chain: {
+      type: "string",
+      desc: "Chain to get the balance for. If not provided the balance for all chains is returned.",
+    },
+  });
+
+type AccountBalance = {
+  chain: string;
+  address: string | undefined;
+  balance: number | undefined;
+};
+
+async function getBalance(
+  chain: string,
+  privateKey: PrivateKey
+): Promise<AccountBalance | undefined> {
+  try {
+    const address = await DefaultStore.chains[chain].getAccountAddress(
```

Comment:
I would put this out of the try/catch and remove the `undefined` possibility from the `address` field in `AccountBalance`. It's better to crash if we can't get the address based on private key 😅 and I don't think the cosmwasm library uses any network to do this, it's just async 🤷

Reply:
Did it.